site/app/libraries/database/DatabaseQueries.php

```diff
@@ -168,7 +168,7 @@ public function getUserByIdOrNumericId($id) {
      */
```

Comment:
Do we need to run the formats in this file or can we just live with passing in the `\DateTime` object directly?

The places you use format:
1. `getPastQueue`
2. `getNumberAheadInQueueThisWeek`
3. `getNumberAheadInQueueToday`

Reply:
The reason I use format sometimes in this file is because that is equivalent to `CURRENT_DATE` while passing in a date is equivalent to `current_timestamp`

`CURRENT_DATE` does not have any hours/min so it rounds to midnight which is what I want. I use that for cutoff to know if somebody has been in the queue today or not. If I used the current time it would include the hour/min which would mean that instead of checking if somebody has been in the queue today, it would instead check if they have been in the queue in the last 24 hours which is not what we want.

Reply:
Ah, alright. You'll probably want to do then `->format('Y-m-d 00:00:00O')` so that we're explicit here on the timezone of the server, and not letting postgres put an arbitrary value for it.